dawn/renderer/utils/sagas.js

```diff
@@ -187,10 +187,10 @@ const _timestamps = [0, 0, 0, 0];
 
 function _needToUpdate(newGamepads) {
   return _.some(newGamepads, (gamepad, index) => {
-    if (!_.isUndefined(gamepad) && (gamepad.timestamp > _timestamps[index])) {
+    if (gamepad !== null && (gamepad.timestamp > _timestamps[index])) {
       _timestamps[index] = gamepad.timestamp;
       return true;
-    } else if (_.isUndefined(gamepad) && !_.isNull(_timestamps[index])) {
```

Comment:
not a big deal, but you can also just get away with checking `if (gamepad && ...)`
assuming `typeof gamepad === 'object'` (and I'd be pretty worried if that weren't always true)